on-chain/contracts/TAParser.sol

```diff
@@ -1,4 +1,4 @@
-// SPDX-License-Identifier: UNLICENSED
+//  SPDX-License-Identifier: MIT
```

Comment:
Without this line hardhat throws warning. So I updated to MIT, since that's what we're using in the repo